mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -105,6 +108,15 @@ class GaussianMixtureModel private[ml] (
   def gaussians: Array[MultivariateGaussian] = parentModel.gaussians
 
   @Since("2.0.0")
```

Comment:
@jkbradley I changed the gaussians return value as DataFrame and it works for python side. Now I change the related scala side which directly uses gaussians. There are two places in the test and one place in the saveImpl function. I will submit the code for review. I think if we can keep two methods, one is the original gaussians, the other is the Dataframe API for Python, the code could be simpler. Now, I just use one method.